### DISCUSSION TITLE
Add Help on how to  “reorder” to sequential playlist

### DISCUSSION THREAD

**Discussion by batsonjay** [AzuraCast/AzuraCast#6833] [Ideas]
As of the time of rolling release 2023-09-29, the way to reorder tracks n a sequential playlist is to use the button visible by selecting "more" on the line for the desired playlist on /station/x/playlist, then clicking "reorder" from the set of buttons that appear.

Because that latter button isn't normally revealed, it can obscure this capability.

As a playlist editor I want to know how to reorder a sequential playlist from AzuraCast documentation on Playlists.

**Comment by Moonbase59** [comment]
Good thinking. This should end up in the docs, too.

See also my feature request [Add "move to top/bottom" buttons to sequential playlist re-arrange window](https://features.azuracast.com/suggestions/518314/add-move-to-topbottom-buttons-to-sequential-playlist-rearrange-window) (awaiting approval).

**Reply by changlee2** [reply]
I think the latest version is more comfortable to reordering tracks.
